web/src/components/Header.tsx

```diff
@@ -72,9 +74,14 @@ export const Header = ({
   const { rect: headerRect, ref } = useElementRect()
   const height = headerRect?.height ?? 0
   const landingPath = pathnameFromRouteInformation({ route: LANDING_ROUTE, languageCode: language })
+  const { xsmall, small } = useDimensions()
 
   return (
-    <Headroom scrollHeight={HEADER_HEIGHT} height={height} zIndex={2} onStickyTopChanged={onStickyTopChanged}>
+    <Headroom
+      scrollHeight={HEADER_HEIGHT}
+      height={height}
+      zIndex={xsmall || small ? 2 : Z_INDEX_ABOVE_SIDEBAR} // 2 for mobile bottom sheet
```

Comment:
🔧 
```suggestion
      // Position header above toolbar but below bottom sheet
      zIndex={mobile ? 2 : Z_INDEX_ABOVE_SIDEBAR}
```

❓ Actually, could you check if the z-index for the toolbar is still necessary? I actually don't think it still is since it was added for the sharing popup which will was removed in #3578.
Then you can just revert this:
``` suggestion
      zIndex={2}
```

Reply:
Ok 🙂